src/test/java/org/folio/dew/batch/bursarfeesfines/service/BursarFilterEvaluatorTest.java

```diff
@@ -0,0 +1,305 @@
+package org.folio.dew.batch.bursarfeesfines.service;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.math.BigDecimal;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.UUID;
+import lombok.extern.log4j.Log4j2;
+import org.folio.dew.domain.dto.Account;
+import org.folio.dew.domain.dto.BursarExportFilter;
+import org.folio.dew.domain.dto.BursarExportFilterAge;
+import org.folio.dew.domain.dto.BursarExportFilterAmount;
+import org.folio.dew.domain.dto.BursarExportFilterCondition;
+import org.folio.dew.domain.dto.BursarExportFilterFeeFineOwner;
+import org.folio.dew.domain.dto.BursarExportFilterFeeType;
+import org.folio.dew.domain.dto.BursarExportFilterLocation;
+import org.folio.dew.domain.dto.BursarExportFilterNegation;
+import org.folio.dew.domain.dto.BursarExportFilterPass;
+import org.folio.dew.domain.dto.BursarExportFilterPatronGroup;
+import org.folio.dew.domain.dto.BursarExportFilterServicePoint;
+import org.folio.dew.domain.dto.Item;
+import org.folio.dew.domain.dto.ItemLocation;
+import org.folio.dew.domain.dto.User;
+import org.folio.dew.domain.dto.bursarfeesfines.AccountWithAncillaryData;
+import org.junit.jupiter.api.Test;
+import org.openapitools.jackson.nullable.JsonNullable;
+
+@Log4j2
+public class BursarFilterEvaluatorTest {
+
+  @Test
+  void evaluateJSONNullableExportFilter() {
+    AccountWithAncillaryData accountWithAncillaryData = AccountWithAncillaryData
+      .builder()
+      .account(null)
+      .user(null)
+      .item(null)
+      .build();
+
+    BursarExportFilterPass bursarExportFilterPass = new BursarExportFilterPass();
+
+    JsonNullable<BursarExportFilter> jsonNullableFilterPass = JsonNullable.of(
+      bursarExportFilterPass
+    );
+
+    assertTrue(
```

Comment:
Either add messages here (to satiate sonar), or use something like hamcrest matchers.  I added a commit to convert this to hamcrest `assertThat` — these add a whole lot more possibilities for matching and I highly recommend using them instead of the more basic assertTrue/assertFalse